app/bundles/CoreBundle/Helper/PageHelper.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright   2020 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        https://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Helper;
+
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+final class PageHelper implements PageHelperInterface
```

Comment:
I really like that you did this instead of doing it in each controller. However I don't get:
1. How is it related to PHP 7.4?
2. Why was this used only in some controllers?

Reply:
I have no clue, this commit was done by you AFAIK :joy: https://github.com/mautic/mautic/pull/8447/commits/8ab8a2b6c35e37c4a0eea00924541dcb665486cb. I just built further on your already existing PR. Probably this somehow got combined with another PR?

Reply:
Haha. OK. My poor memory. 🤦  So the question is why I did it :D Probably some small change I had to do in multiple controllers and decided to refactor it and make the change only in one place instead.